PayButton/paybutton-server
# [Issue #349] feat: sync addresses' unconfirmed txs

Description:
Part of #349. When initially syncing existent addresses, also looks for unconfirmed txs and syncs them.

Test plan:
1. Stop the automatic syncing of addresses with `yarn docker js`
2. Create a paybutton with one of your addresses
3. Send a tx to it
4. Before it gets confirmed, sync it with `curl http://localhost:3000/address/transactions/sync/[youraddress]`. The output should display this transaction identifiable by the `confirmed: false` property.
6. It should also appear on the button detail view, also with the `confirmed` column set to `no`.

Depends on:
- [x] #350
